src/helpers/promise.js

```diff
@@ -1,4 +1,5 @@
 /* eslint no-proto:0 */
+/* eslint no-use-before-define:0 */
```

Comment:
I think in the `.eslintrc` is better, in this case.  I generally prefer globals there, otherwise they are easier to lose track of.